src/Windows/R/Components/Impl/InteractiveWorkflow/Implementation/RInteractiveWorkflow.cs

```diff
@@ -23,16 +22,14 @@
 using Microsoft.VisualStudio.R.Package.Repl;
 
 namespace Microsoft.R.Components.InteractiveWorkflow.Implementation {
-    public sealed class RInteractiveWorkflow : IRInteractiveWorkflow {
+    public sealed class RInteractiveWorkflow : IRInteractiveWorkflowVisual {
```

Comment:
I don't think we should rename this interface at all, cause there is nothing "visual" in this class, even in its implementation. It references `IInteractiveWindowVisualComponent` in these 3 methods:
```cs
        IInteractiveWindowVisualComponent ActiveWindow { get; }
        event EventHandler<ActiveWindowChangedEventArgs> ActiveWindowChanged;
        Task<IInteractiveWindowVisualComponent> GetOrCreateVisualComponentAsync(int instanceId = 0);
```
but it can be decoupled later if needed.